readme.html

```diff
@@ -1,29 +1,468 @@
+            <li>Also store the Base URL and port from the URL.</li>
+            <li>Double click on the selected API key to get the Org ID</li>
+        </ul>
+        <li>Log in to your Phantom platform.</li>
+        <ul>
+            <li>Navigate to the <b>Home</b> dropdown and select <b>Apps</b>.</li>
+            <li>Search the Illumio App from the search box.</li>
+            <li>Click on the <b>CONFIGURE NEW ASSET</b> button.</li>
+            <li>Navigate to the <b>Asset Info</b> tab and enter the Asset name and Asset description.</li>
+            <li>Navigate to the <b>Asset Settings</b> tab.</li>
+            <li>Paste the <b>API Key, API Secret, Hostname, Port, and Org ID</b> from Illumio to its respective
+                configuration parameter.</li>
+            <li>Save the asset.</li>
+            <li>Now, test the connectivity of the Phantom server to the Illumio instance by clicking on the <b>TEST
+                    CONNECTIVITY</b> button.</li>
+        </ul>
+    </ul>
+    </p>
+    <h2>Explanation of the Asset Configuration Parameters</h2>
+    The asset configuration parameters affect 'test connectivity' and some other actions of the application. The
+    parameters related to test connectivity action are Illumio API Key, API Secret, Hostname, Port and Org ID.
+    <ul>
+        <li>
+            <b>API Key: </b> Illumio Authentication Username for asset authorization.
+        </li>
+        <li>
+            <b>API Secret</A>: </b> API Secret for asset authorization.
+        </li>
+        <li>
+            <b>Hostname</A>: </b> Name of host where PCE is hosted.
+        </li>
+        <li>
+            <b>Port</A>: </b> Port number where PCE is hosted.
+        </li>
+        <li>
+            <b>Org ID</A>: </b> ID number of organization in PCE.
+        </li>
+    </ul>
+    </p>
+    <p>
+    <h2>Explanation of the Illumio Actions' Parameters</h2>
+    <ul>
+        <li>
+            <h3>Test Connectivity (Action Workflow Details)</h3>
+            <ul>
+                <li> This action will test the connectivity of the Phantom server to the Illumio instance by using the
+                    provided asset configuration parameters. </li>
+                <li> The action validates the provided asset configuration parameters. Based on the response from the
+                    SDK method, the appropriate success and failure message will be displayed when the
+                    action gets executed.</li>
+            </ul>
+        </li>
+        <li>
+            <h3>Get Traffic Analysis</h3>
+            Runs an explorer query to get a traffic analysis report based on the provided inputs. The query checks all
+            sources and destinations for traffic on a given port.
+            <ul>
+                <li><b>Action Parameter: Start Time</b></li>
+                <ul>
+                    <li>This parameter will accept the start datetime for a traffic. An error message will be
```

Comment:
Add notes that start and end times support human readable or ISO datetime values

Reply:
Added note